AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_CubeMapCaptureAdded.py

```diff
@@ -14,10 +14,27 @@ class Tests:
         "P0: Entity failed to find CubeMap Capture component")
     cubemap_capture_component_removal = (
         "UNDO CubeMap Capture Entity success",
-        "UNDO CubeMap Capture Entity failed")
+        "P0: UNDO CubeMap Capture Entity failed")
     removal_undo = (
         "REDO CubeMap Capture Entity success",
-        "REDO CubeMap Capture Entity failed")
+        "P0: REDO CubeMap Capture Entity failed")
+
```

Comment:
blank lines between declared tuples are not in line with the style we are using for these.

Reply:
updated, thanks for pointing that out.